spec/requests/properties_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe PropertiesController, type: :request do
+  describe 'POST properties' do
+    let(:assessment) { create :assessment }
+    let(:property) { create :property, assessment: assessment }
+    let(:dummy_payload) { { key: :value }.to_json }
+
+    context 'valid params' do
+      before do
+        expect(PropertiesCreationService).to receive(:call).with(dummy_payload).and_return(success_response)
+        post assessment_properties_path(assessment), params: dummy_payload
+      end
+
+      it 'returns http status code 200' do
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'returns the expected success response payload' do
+        expect(response.body).to eq success_response.to_h.to_json
+      end
+    end
+
+    context 'invalid params' do
+      before do
+        expect(PropertiesCreationService).to receive(:call).with(dummy_payload).and_return(error_response)
+        post assessment_properties_path(assessment), params: dummy_payload
+      end
+
+      it 'returns http status code 422' do
+        expect(response).to have_http_status(:unprocessable_entity)
+      end
+
+      it 'returns the expected error response payload' do
+        expect(response.body).to eq error_response.to_h.to_json
+      end
+    end
+  end
+
+  def success_response
```

Comment:
you should use the `let` keyword here rather than a method. To follow RSpec's convention